docs_app/src/styles/1-layouts/_api-page.scss

```diff
@@ -1,88 +1,93 @@
 .api-section {
-    position: relative;
-
-    pre {
```

Comment:
This file should be reviewed with whitespace changes [turned off](https://github.com/ReactiveX/rxjs/pull/7027/files?diff=split&w=1).